src/App.css

```diff
@@ -1,5 +1,5 @@
 .App {
-	background-color: hsl(257, 40%, 49%);
+	background-color: #000;
 	font-family: 'Open Sans', sans-serif;
 	color: white;
```

Comment:
Be consistent with how you use colors. If you use hex codes, use hex codes, but avoid hex codes with named colors like line 4.